src/legacy/proza-message/test/unit/vl-proza-message.test.html

```diff
@@ -33,533 +33,209 @@
-
-      /*test('zal een class toevoegen wanneer het block attribuut aanwezig is', () => {
-        const proza = fixture('vl-proza-message-fixture');
-        assert.isFalse(proza.classList.contains('vl-proza-message__block'));
-        proza.setAttribute('data-vl-block', '');
-        assert.isTrue(proza.classList.contains('vl-proza-message__block'));
-        proza.removeAttribute('data-vl-block');
-        assert.isFalse(proza.classList.contains('vl-proza-message__block'));
-      });*/
-
-      test('zal op basis van de inhoud van het proza bericht het block attribuut automatisch toevoegen indien nodig', (done) => {
-        const construct = (message, callback) => {
-          delete VlProzaMessage.__cache;
-          delete VlProzaMessagePreloader.__cache;
-          const proza = fixture('vl-proza-message-fixture');
-          fetchMock.get('proza/domein/foo/bar', {code: 'bar', tekst: message}, {overwriteRoutes: true});
-          fetchMock.flush(true).then(() => awaitUntil(() => proza._wysiwygElement.innerHTML === message).then(() => callback(proza)));
-        };
-
-        construct('message', (proza) => {
-          assert.isNull(proza.getAttribute('data-vl-block'));
-          construct('<span>message</span>', (proza) => {
-            assert.isNull(proza.getAttribute('data-vl-block'));
-            construct('<p>message</p>', (proza) => {
-              assert.isNotNull(proza.getAttribute('data-vl-block'));
-              construct('<table></table>', (proza) => {
-                assert.isNotNull(proza.getAttribute('data-vl-block'));
-                construct('<span style="display: block;">message</span>', (proza) => {
-                  assert.isNotNull(proza.getAttribute('data-vl-block'));
-                  construct('<span style="display: inline-block;">message</span>', (proza) => {
-                    assert.isNotNull(proza.getAttribute('data-vl-block'));
-                    construct('<span style="display: flex;">message</span>', (proza) => {
-                      assert.isNotNull(proza.getAttribute('data-vl-block'));
-                      construct('<span style="display: grid;">message</span>', (proza) => {
-                        assert.isNotNull(proza.getAttribute('data-vl-block'));
-                        done();
-                      });
-                    });
-                  });
-                });
-              });
-            });
+          test('bij het drukken op de refresh knop, word de nieuwste proza-message ophaalt', async () => {
+            const proza = fixture('vl-proza-message-fixture');
+            await fetchMock.flush(true);
+            await awaitUntil(() => proza._typographyElement.textContent === 'foobar');
+            fetchMock.get('proza/domein/foo/bar', {code: 'bar', tekst: 'barfoo'}, {overwriteRoutes: true});
+            await fetchMock.flush(true);
+            proza._refreshButton.click();
+            await awaitUntil(() => proza._typographyElement.textContent === 'barfoo');
```

Comment:
checken dat het daarna in de cache(s) gewijzigd is, dus dat als je daarna de message ophaalt dat je wel degelijk de nieuwe krijgt, niet alleen op het scherm in de typography, maar ook bij het ophalen van de message dus